shared/js/dataDriver.js

```diff
@@ -275,6 +275,22 @@ loop.DataDriver = function() {
     }
 
     /**
+     * Request all room data except for chat.
+     *
+     * @return {Promise} Resolved with an {Object} with keys of record ids.
+     */
+    requestRoomData() {
+      let query = this._buildQuery({
+        orderBy: '"$key"',
+        // NB: All record types come lexicographically after chat.
```

Comment:
Is this an invariant that we need to maintain when adding new record types?  If so, how can we ensure that that happens?